examples/compose/docker-compose.yml

```diff
@@ -104,6 +104,20 @@ services:
     image: vitess/lite:${VITESS_TAG:-latest}
     volumes:
     - .:/script
+  set_keyspace_durability_policy:
```

Comment:
I set a rather specific name for now.
I think it's appropriate to change the name later when you need to add other commands at this point.